backend/hct_mis_api/apps/grievance/services/needs_adjudication_ticket_services.py

```diff
@@ -216,34 +231,37 @@ def mark_as_duplicate_individual_and_reassign_roles(
     ticket_details: TicketNeedsAdjudicationDetails,
     individual_to_remove: Individual,
     user: AbstractUser,
-    unique_individual: Individual,
+    need_reassign_roles: bool = True,
+    unique_individual: Optional[Individual] = None,
 ) -> None:
+    household = individual_to_remove.household
     if ticket_details.is_multiple_duplicates_version:
-        household = reassign_roles_on_disable_individual_service(
-            individual_to_remove,
-            ticket_details.role_reassign_data,
-            user,
-            ticket_details.ticket.programs.all(),
-            "new_individual",
-        )
+        if need_reassign_roles:
+            reassign_roles_on_disable_individual_service(
+                individual_to_remove,
+                ticket_details.role_reassign_data,
+                user,
+                ticket_details.ticket.programs.all(),
+                "new_individual",
+            )
     else:
-        household = reassign_roles_on_disable_individual_service(
-            individual_to_remove, ticket_details.role_reassign_data, user, ticket_details.ticket.programs.all()
-        )
+        if need_reassign_roles:
```

Comment:
elif

Reply:
I will update it in separate PR later today or tomorrow morning